lib/pages/settings_pages/settings_page.dart

```diff
@@ -298,12 +296,25 @@ class _SettingsPageState extends State<SettingsPage> {
     );
   }
 
+  languagesSortedAlphabetically() {
+    // Alphabet depends on the currently selected language
+    List<Language> sortedLanguages = languageCodes.map((language) {
+      String translatedName = MyLocalizations.of(context, language.isoCode) ?? language.name;;
```

Comment:
Change double `;;` to single `;` at the end of the line

Reply:
sorry for the typo!